src/components/layout/FeaturedDomains/FeaturedDomains.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+
+import { ArNSDomains } from '../../../types';
+import { ArnsCard } from '../../cards';
+import './styles.css';
+
+function FeaturedDomains(props: { domains: ArNSDomains }) {
+  const { domains } = props;
+
+  // show three at a time by default
+  const DEFAULT_INCREMENT = 3;
```

Comment:
might be worth also setting `MAX_COUNT` so only up to like DEFAULT_INCREMENT * 3 are displayed